### PR TITLE
Added changelog, prepared release

### PR DESCRIPTION
**Pull request by MarleneKress79789**

### CODE REVIEW

**Review by tkilias — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+### All Submissions:
+
+* [ ] Is the title of the Pull Request correct?
```
Comment (ckunki) [inline .github/pull_request_template.md:3]:
```suggestion
* [ ] Is the title of the pull request correct?
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+* [ ] Is the title of the corresponding issue correct?
+* [ ] Have you updated the changelog?
+* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../../pulls) for the same update/change? <!-- markdown-link-check-disable-line -->
```
Comment (ckunki) [inline .github/pull_request_template.md:6]:
```suggestion
* [ ] Have you checked to ensure there aren't other open [pull requests](../../../../pulls) for the same update/change? <!-- markdown-link-check-disable-line --> 
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+### All Submissions:
+
+* [ ] Is the title of the Pull Request correct?
+* [ ] Is the title of the corresponding issue correct?
+* [ ] Have you updated the changelog?
+* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../../pulls) for the same update/change? <!-- markdown-link-check-disable-line --> 
+* [ ] Are you mentioning the issue which this PullRequest fixes ("Fixes...")
```
Comment (ckunki) [inline .github/pull_request_template.md:7]:
```suggestion
* [ ] Are you mentioning the issue which this pull request fixes ("Fixes...")
```

File: README.md [hunk 4]
```diff
@@ -14,15 +14,15 @@ In any case, you need to verify your UDFs with integrations test inside the data
...
+pip install udf-mock-python
 ```
 
 ### Installing via poetry
 Add it to your `tool.poetry.dependencies` or `tool.poetry.dev-dependencies`
 
 ```
 [tool.poetry.dev-dependencies]
-exasol-udf-mock-python = { git = "https://github.com/exasol/udf-mock-python.git", branch = "main" }
+udf-mock-python = = "^0.1.0"
```
Comment (ckunki) [inline README.md:25]:
```suggestion
udf-mock-python = "^0.1.0"
```

File: doc/changes/changes_0.1.0.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+# UDF Mock Python 0.1.0, released 2023.11.21
+
+Code name: Initial Release
+
+## Summary
```
Comment (ckunki) [inline doc/changes/changes_0.1.0.md:5]:
```suggestion
## Summary

```

File: doc/changes/changes_0.1.0.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+This is the initial release of the UDF Mock Python which provides a mock runner for Python3 UDFs which allows you to
```
Comment (ckunki) [inline doc/changes/changes_0.1.0.md:6]:
```suggestion
This is the initial release of the UDF Mock Python providing a mock runner for Python3 UDFs which allows you to
```

**Review by ckunki — CHANGES_REQUESTED** [review]

File: doc/changes/changes_0.1.0.md (hunk 6 above)
Comment (MarleneKress79789) [reply doc/changes/changes_0.1.0.md:6]: i like the other more in this case.

File: doc/changes/changes_0.1.0.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
+# UDF Mock Python 0.1.0, released 2023.11.21
```
Comment (MarleneKress79789) [inline doc/changes/changes_0.1.0.md:1]:
```suggestion
# UDF Mock Python 0.1.0, released 2023.11.22
```

**Review by tkilias — APPROVED** [review]

**Pull request merged**